src/DeviceConnectControl.js

```diff
@@ -11,6 +11,7 @@ import './DeviceConnectControl.css'
 type DeviceConnectControlProps = {
     intl: any,
     children: React.Element<any>, // Button contents
+    bluetoothApiIsAvailable: boolean,
```

Comment:
Could we instead rename this property to "disabled". I'm thinking that a more general disabled property is useful and we don't need to specify the reason why it's disabled in the API. The control doesn't vary based on the reason why it's disabled, only that it's disabled.